support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.jsx

```diff
@@ -146,12 +151,16 @@ function WeeklyCheckoutForm(props: PropTypes) {
           }
           title="Guardian Weekly"
           description=""
-          productPrice={{ price: 99.89, currency: 'GBP' }}
+          productPrice={summaryPrice}
           promotion={undefined}
```

Comment:
We're going to need to pass in the promotion here

Reply:
Yes - as discussed, I'll leave this one until after your weekly pricing rejig is complete.